grails-app/views/report/showInventoryReport.gsp

```diff
@@ -158,8 +158,8 @@
                                 </g:link>
                             </li>
                             <li>
-                                <g:link url="https://github.com/openboxes/openboxes/releases/tag/v${g.meta(name:'app.version')}" target="_blank">
-                                    <warehouse:message code="docs.releaseNotes.label" default="Release Notes"/> (${g.meta(name:'app.version')})
+                                <g:link url="https://github.com/openboxes/openboxes/releases/tag/v${ConfigHelper.getAppVersion()}" target="_blank">
```

Comment:
Again, if possible it would be best to use info.app.version if that's the convention.

Reply:
fixed!